app/controllers/group_assignments_controller.rb

```diff
@@ -21,4 +21,35 @@ def create
       end
     end
   end
+
+  def destroy
+    @meeting = Meeting.find(params["meeting_id"])
+    @individual = Individual.find(params["individual_id"])
+
+    if params["group_assignment"]["group"].empty?
+      flash[:alert] = "Please select a valid group!"
+      redirect_to meeting_individual_path(@meeting, @individual)
+    else
+
+      @group = Group.find(params["group_assignment"]["group"])
```

Comment:
same as above in terms of assuming nil - you want to be defensive when dealing with compound data structures in the controller.